DirectXTex/DirectXTexMipmaps.cpp

```diff
@@ -543,7 +543,7 @@ HRESULT DirectX::Internal::ResizeSeparateColorAndAlpha(
 
         if (SUCCEEDED(hr))
         {
-            WICInProcPointer colorWithAlphaData = nullptr;
```

Comment:
Another type missing from the MinGW headers, but this one is just a typedef for ``BYTE *`` which is a lot clearer anyhow.